CinemaTicketSalesSystem/CinemaTicketSalesSystem/Controllers/HomeController.cs

```diff
@@ -3,27 +3,20 @@
 using System.Linq;
 using System.Web;
 using System.Web.Mvc;
+using ApplicationDbMovies.Contexts;
+using ApplicationDbMovies.Models;
 
 namespace CinemaTicketSalesSystem.Controllers
 {
     public class HomeController : Controller
     {
-        public ActionResult Index()
-        {
-            return View();
-        }
-
-        public ActionResult About()
-        {
-            ViewBag.Message = "Your application description page.";
+        ApplicationDbContext db = new ApplicationDbContext();
```

Comment:
Web layer should not have access to DAL. Please create a service in BL layer and use it here.

It also should be injected with autofac.